aemedge/blocks/cards/cards.js

```diff
@@ -325,16 +326,17 @@ async function fetchAndFilterUpcomingEconodayEvent() {
 
 export async function createDynamicCards(block, numEntries = null) {
   const config = readBlockConfig(block);
+  const tags = getMetadata('article:tag');
+  const tagsArray = tags ? tags.split(',').map((tag) => tag.trim().toLowerCase()) : [];
```

Comment:
if tags coming from the current page are required for a variation, please move this to the specific section in the code.
Please verify if "course" variation is being used in another page where tags comes from block metadata. If yes, you could add a variation like (course, related) to indicate where the tags will come from